services/hanke-service/src/test/kotlin/fi/hel/haitaton/hanke/hakemus/KaivuilmoitusSendValidationTest.kt

```diff
@@ -378,6 +378,35 @@ class KaivuilmoitusSendValidationTest {
 
             assertThat(validateArea(area, "area")).failedWith("area.tyonTarkoitukset")
         }
+
+        @Test
+        fun `fails when a single tyoalue has an index but there's no haittojenhallinta for that index`() {
```

Comment:
Isn't there two tyoalues that have autoliikenne indexes? Not a single tyoalue.

Reply:
Yes, thank you. I changed the test data to match the description. I think miscopied the `withZeroAutoliikenne` and `withPositiveAutoliikenne` before.